ArduCopter/mode_poshold.cpp

```diff
@@ -530,7 +530,7 @@ void ModePosHold::update_brake_angle_from_velocity(float &brake_angle, float vel
     float lean_angle;
     float brake_rate = g.poshold_brake_rate;
 
-    brake_rate /= 4.0f;
+    brake_rate /= (float)LOOP_RATE_FACTOR;
```

Comment:
This appears to be a change in behaviour. Can you explain why you have changed the constant factor of 4.0 to LOOP_RATE_FACTOR?

Reply:
Thank you @lthall 

In @Ju1ien 's original code
https://github.com/Ju1ien/ArdupilotST-JD/blob/90b0cdb40cb0c830ea3a27624984dc71cbe0c168/ArduCopter/control_poshold.pde#L563-L568
4 is LOOP_RATE_FACTOR
And I have asked @Ju1ien about it (Thanks @Ju1ien very much for explaining many details)

I think it may be easier to maintain. So that we will not forget to change it when we change loop rate

Reply:
Yes at the moment the definition says "#define LOOP_RATE_FACTOR 4" so there should be no change in behaviour.